lib/hexpm_web/templates/page/search.html.heex

```diff
@@ -0,0 +1,35 @@
+<form phx-change="suggest" phx-submit="search"
+      role="search"
+      action={Routes.package_path(Endpoint, :index)}
+      method="get">
+  <div class="input-group">
+    <input type="text"
+           list="matches"
+           class="form-control"
+           name="search"
+           id="search"
+           phx-debounce="1500"
+           tabindex="1"
+           value={@query}
+           list="matches"
+           placeholder="Start typing..." {%{readonly: @loading}}/>
```

Comment:
Does this means it is rejecting input while it's loading? I don't think we want that.

Reply:
Removed.